app/controllers/admin/api_docs/base_controller.rb

```diff
@@ -12,6 +12,7 @@ def index
 
   def new
     @api_docs_service = api_docs_services.new
+    @commit_button_label = 'Create Spec'
```

Comment:
Do we need an instance variable in the controller for this? This either goes to the view directly or at least a helper method that returns 'Create Spec' or 'Update Spec' depending on the `action_name` for the view to use